README.md

```diff
@@ -14,13 +14,32 @@ TODO
 
 \subsubsection*{Cantor set}
 \subsubsection*{Cartesian space ${\mathbb R}^n$}
+The set of all $n$-tuples, $\{ (x_1, x_2, ... , x_n) : x_i \in \mathbb{R} \}$.\\
```

Comment:
I do
`\langle x_1, x_2, ... , x_n\rangle \mid x_i \in {\mathbb R}`
but it's debatable

Reply:
Yeah, either is cool, I'm just used to algebra where angle brackets represent the subobject generated by those elements.